package.json

```diff
@@ -85,6 +85,7 @@
     "build": "webpack",
     "watch": "webpack --watch",
     "package": "webpack --mode production --devtool hidden-source-map",
+    "package:prerelease": "npx vsce package --pre-release",
```

Comment:
This script looks like it publishes a pre-release version of a package. Could you help me understand the general workflow here and how it relates to this issue?

Reply:
This will build the project and output a `.vsix` file which makes it easier to test (or send to someone else to test).

Eventually, I'd like to a GitHub workflow to build on PRs and upload the `.vsix` as an artifact to make it easier to test. Not directly related to this PR but should help for future troubleshooting!